### DISCUSSION TITLE
Fehlender Code des Lieferlandes bei TaxCategory = 'K', wenn Lieferanschrift = Erwerberanschrift ist

### DISCUSSION THREAD

**Discussion by dsshad** [LandrixSoftware/XRechnung-for-Delphi#13] [Q&A]
Hallo,
bei einer umsatzsteuerbefreiten Rechnung (TaxCategory = idtfcc_K_VATExemptForEEAIntracommunitySupplyOfGoodsAndServices) zeigt der Validator diesen Fehler an:

> [BR-IC-12]-In an Invoice with a VAT breakdown (BG-23) where the VAT category code (BT-118) is "Intra-community supply" the Deliver to country code (BT-80) shall not be blank. 
Pfad: /rsm:CrossIndustryInvoice/rsm:SupplyChainTradeTransaction[1]/ram:ApplicableHeaderTradeSettlement[1]/ram:ApplicableTradeTax[1]/ram:CategoryCode[1] 

BT-80 ist Teil der Gruppe "DELIVER TO ADDRESS". Die Gruppe ist nur zu verwenden, wenn die Lieferanschrift von der Erwerberanschrift abweicht. Im aktuellen Beispiel ist es nicht der Fall, also wird die Lieferanschrift nicht eingetragen.

Wie handhabt man das dann?

Ediz: Ausgabeformat = ZUGFeRD.

**Comment by landrix** [comment]
Dann sollte die Lieferanschrift mit ausgegeben werden. Siehe auch folgendes Beispiel:

https://github.com/LandrixSoftware/XRechnung-for-Delphi/blob/ee6ac49724d7ab87cf69d4a1c0a8d6989ebe758c/Samples/XRechnungUnit2TestCases.pas#L719
